common/scala/src/main/scala/org/apache/openwhisk/core/connector/Message.scala

```diff
@@ -470,6 +470,34 @@ case class InvokerResourceMessage(status: String,
    * Serializes message to string. Must be idempotent.
    */
   override def serialize: String = InvokerResourceMessage.serdes.write(this).compactPrint
+
+  def canEqual(a: Any) = a.isInstanceOf[InvokerResourceMessage]
+
+  override def equals(that: Any): Boolean =
+    that match {
+      case that: InvokerResourceMessage => {
+        that.canEqual(this) &&
```

Comment:
I think `isInstanceOf` is not required after using the case match?

Reply:
Good.
Updated accordingly.